pkg/utils/gardener/shoot.go

```diff
@@ -279,6 +280,13 @@ func IsShootProjectInternalSecret(secretName string) (string, bool) {
 	return "", false
 }
 
+// ComputeManagedShootIssuerSecretName returns the name that should be used for
+// storing the service account public keys of a shoot's kube-apiserver
+// in the gardener-system-shoot-issuer namespace in the Garden cluster.
+func ComputeManagedShootIssuerSecretName(projectName string, shootUID types.UID) string {
+	return fmt.Sprintf("%s--%s", projectName, shootUID)
```

Comment:
What about using the shoot technical id similar to backupentries so that it is easier for operators to quickly see to which shoot this secret is related?

Reply:
As per GEP 24 the url that will serve the shoot's openid discovery documents will contain two path parameters - the project name and the shoot UID (ex `/projects/myproj/shoots/f924a208-1034-4aeb-84d9-b0184894a0cf/issuer`). Depending on the implementation of the discovery/metadata server this can result in a direct call to the API server 
```
k get secret projectParam--uidParam
```

or that can result in a cache entry search
```
cache.Get("projectParam--uidParam")
```

In addition to that labels indicating the shoot namespace and shoot name are added to the secret.